.storybook/main.ts

```diff
@@ -25,6 +25,7 @@ const config: StorybookConfig = {
     getAbsolutePath('@storybook/addon-onboarding'),
     getAbsolutePath('@storybook/addon-interactions'),
     getAbsolutePath('@storybook/addon-a11y'),
+    getAbsolutePath('@storybook/addon-designs'),
```

Comment:
Where is the old plugin being removed?  Or is that planned for a followup PR?

Reply:
The other one wasn't a plugin-- it's part of Chromatic, and I don't think we did anything to enable it.